### PR TITLE
[vmdk api adoption] Fix 2 bugs

### PR DESCRIPTION
**Pull request by dntczdx**
1. In PR #1754 , "startTime := time.Now()" before the daisy workflow running got removed, so the inflation time didn't get calculated correwctly. Bring it back. Otherwise, the inflation time will be always 0.
2. The gcsfuse was used without arg "--implicit-dirs". Added it. This bug made the file inspection never taking effect. The file inspection works fine on local without gcloud wrapper, because it avoided the nested folder "tmpimage". Once we call it via gcloud, the nested folder is always involved, so the bug was always triggered.

### CODE REVIEW

**Review by EricEdens — APPROVED** [review]

File: cli_tools/common/gcsfuse/client.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func (client defaultClient) MountToTemp(ctx context.Context, bucket string) (str
 	if err != nil {
 		return "", fmt.Errorf("failed to create a destination directory: %w", err)
 	}
-	cmd := exec.CommandContext(ctx, "gcsfuse", bucket, dir)
+	cmd := exec.CommandContext(ctx, "gcsfuse", "--implicit-dirs", bucket, dir)
```
Comment (EricEdens) [inline cli_tools/common/gcsfuse/client.go:46]: Can you add some unit tests that confirm the bug, and then show that your fix address the isssue?

File: cli_tools/common/image/importer/daisy_inflater.go [hunk 2]
```diff
@@ -72,7 +73,7 @@ func (inflater *daisyInflater) Inflate() (persistentDisk, inflationInfo, error)
 			sourceType: serialValues[importFileFormatKey],
 		}, inflationInfo{
 			checksum:      serialValues[diskChecksumKey],
-			inflationTime: time.Since(time.Now()),
+			inflationTime: time.Since(startTime),
```
Comment (EricEdens) [inline cli_tools/common/image/importer/daisy_inflater.go:76]: ditto on unit tests.

Comment (dntczdx) [reply cli_tools/common/image/importer/daisy_inflater.go:76]: done

File: cli_tools/common/gcsfuse/client.go (hunk 1 above)
Comment (EricEdens) [reply cli_tools/common/gcsfuse/client.go:46]: Talked offline, and Tony pointed out that gcsfuse would be difficult for unit tests. I suggested creating a module test similar to [OS inspect](https://github.com/GoogleCloudPlatform/compute-image-tools/blob/5bd10d7493e35f6fb3ab3ed440f56ece695568b4/cli_tools_tests/module/diskinspect/disk_inspect_test.go#L341-L340)

In the test, call `NewGCSInspector`, then run `Inspect` with a gcs path, and assert the results are what you'd expect.

Comment (dntczdx) [reply cli_tools/common/gcsfuse/client.go:46]: added a new file for that

**Review by EricEdens — APPROVED** [review]
looks great!

**Review by zoran15 — APPROVED** [review]

**Pull request merged**